common/it/com/commercetools/sunrise/categorytree/CategoriesWithProductCountQueryIntegrationTest.java

```diff
@@ -0,0 +1,65 @@
+package com.commercetools.sunrise.categorytree;
+
+import com.commercetools.sunrise.it.WithSphereClient;
+import io.sphere.sdk.categories.queries.CategoryQueryModel;
+import io.sphere.sdk.products.ProductDraft;
+import io.sphere.sdk.queries.PagedQueryResult;
+import io.sphere.sdk.queries.QueryPredicate;
+import io.sphere.sdk.queries.QuerySort;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.function.Consumer;
+
+import static com.commercetools.sunrise.it.CategoryTestFixtures.categoryDraft;
+import static com.commercetools.sunrise.it.CategoryTestFixtures.withCategory;
+import static com.commercetools.sunrise.it.ProductTestFixtures.*;
+import static com.commercetools.sunrise.it.ProductTypeTestFixtures.productTypeDraft;
+import static com.commercetools.sunrise.it.ProductTypeTestFixtures.withProductType;
```

Comment:
Having a public sdk test lib would really help here... But great that you created an integration test 👍